scripts/codebuild/install.sh

```diff
@@ -19,12 +19,15 @@ echo "${magenta}----- CUSTOMIZATIONS -------${reset}"
 ####  content/*
 
 ## get environment variable from parameter store.
+## setting stativ variables for now
+client="https://search-super-testy-search-test-xweemgolqgtta6mzqnuvc6ogbq.us-east-1.es.amazonaws.com"
+site="../../site"
 
 pushd scripts/gatsby-source-iiif
```

Comment:
Since this and the code in generate.sh are the same should we use generate here instead?